client/src/components/ModalDialog/ModalDialog.module.css

```
@@ -0,0 +1,44 @@
+.modalWrapper.displayed {
+  display: block;
+}
+
+.modalWrapper.hidden {
+  display: none;
+}
+
+.scrim {
+  position: fixed;
+  z-index: 9999;
+  top: 0;
+  left: 0;
+  width: 100%;
+  height: 100%;
+  pointer-events: all;
+  background: rgba(0, 0, 0, 0.25);
+}
+
+.modal {
+  background: white;
+  position: absolute;
+  top: 45%;
+  left: 50%;
+  transform: translate(-50%, -50%);
+  padding: 40px 80px;
+  border-radius: 10px;
+  box-shadow: 0 10px 40px -20px rgba(0, 0, 0, 0.25);
+}
+
+.modalWrapper .modal > :global(button) {
+  position: absolute;
+  top: 18px;
+  right: 28px;
+  padding: 3px;
```

```suggestion
  top: 12px;
  right: 12px;
  padding: 0;
```

Should be 12px according to #74.